mesa/visualization/ModularVisualization.py

```diff
@@ -240,17 +240,22 @@ class ModularServer(tornado.web.Application):
     verbose = True
 
     port = int(os.getenv("PORT", 8521))  # Default port to listen on
+    urlpath = str(os.getenv("URLPATH", ""))
```

Comment:
Just to be sure, this `urlpath` variable needs to have the trailing `/` in the end to be stripped, otherwise it becomes `/urlpath//local/...` later on.